modules/cudaoptflow/src/nvidiaOpticalFlow.cpp

```diff
@@ -249,437 +239,6 @@ class LoadNvidiaModules
     PFNNvOFAPICreateInstanceCuda GetOFLibraryFunctionPtr() { return m_NvOFAPICreateInstanceCuda; }
 };
 
-class NvidiaOpticalFlowImpl : public cv::cuda::NvidiaOpticalFlow_1_0
```

Comment:
I believe `1_0` implementation can work with `2.0` SDK.
Perhaps there is no reason to remove that.

Reply:
Yes, you were right.
I could restore the `1_0` implementation.

There was another build failure but that was due to the wrong function call in the disabled part.